lib/msal-node/docs/certificate-credentials.md

```diff
@@ -104,25 +95,13 @@ z2HCpDsa7dxOsKIrm7F1AtGBjyB0yVDjlh/FA7jT5sd2ypBh3FVsZGJudQsLRKfE
 -----END ENCRYPTED PRIVATE KEY-----
 ```
 
-> :information_source: Alternatively, your private key may begin with `-----BEGIN PRIVATE KEY-----` (pkcs#12) or `-----BEGIN RSA PRIVATE KEY-----` (pkcs#1). These formats are also permissible.
-
-If you have encrypted your private key (or if your private key is already encrypted) with a *pass phrase* as recommended, you'll need to decrypt it before passing to **MSAL Node**. This can be done using Node's [crypto module](https://nodejs.org/docs/latest-v12.x/api/crypto.html):
-
-First, ensure that your private key is of type `pkcs8`:
-
-```bash
-openssl pkcs8 -topk8 -inform PEM -outform PEM -in example.key -out example.key
-```
-
-You'll be prompted for the pass phrase you specified earlier and a new encryption password:
-
-```console
-Enter pass phrase for example.key.pem:
-Enter Encryption Password:
-Verifying - Enter Encryption Password:
-```
+> :information_source: Alternatively, your private key may begin with `-----BEGIN PRIVATE KEY-----` (unencrypted *PKCS#8*) or `-----BEGIN RSA PRIVATE KEY-----` (*PKCS#1*). These formats are also permissible. If you need, you can ensure that your private key is in *PKCS#8* format by typing:
```

Comment:
Consider:  "The following can be used to convert any compatible key to the *PKCS#8* key type.

Reply:
done